src/serializers/index.ts

```diff
@@ -77,14 +80,26 @@ const res = (response: Response) =>
       }
     : response;
 
-export const createSerializers = (opts: SerializerOptions) => {
+const createErrSerializer =
+  (
+    serializer: (error: Error) => SerializedError,
+    opts: FormatterOptions,
+  ): SerializerFn =>
+  (error: unknown): unknown =>
+    trimmer({
+      depth: opts.maxObjectDepth ?? 4,
+    })(serializer(error as Error));
+
+export const createSerializers = (
```

Comment:
I like @samchungy's question of whether we should apply this to all serialisers. Would all also include user-provided ones? 🤔